src/App/Components/Layout/Footer/toggle-settings.jsx

```diff
@@ -32,9 +32,9 @@ const ToggleSettings = ({
                 in={is_settings_visible}
                 timeout={250}
                 classNames={{
-                    enter    : 'settings-modal__container--enter',
-                    enterDone: 'settings-modal__container--enter-done',
-                    exit     : 'settings-modal__container--exit',
+                    enter    : 'modal__container--enter',
+                    enterDone: 'modal__container--enter-done',
+                    exit     : 'modal__container--exit',
                 }}
                 unmountOnExit
             >
```

Comment:
Can you add `appear` prop because `enter` transitions don't seem to work on initial mount. Please add the `className` for `appear` as well, and you can use the same `enter` modifier class.
```
<CSSTransition
      appear
      in={is_settings_visible}
      timeout={250}
      classNames={{
          appear   : 'modal__container--enter',
          enter    : 'modal__container--enter',
          enterDone: 'modal__container--enter-done',
          exit     : 'modal__container--exit',
      }}
      unmountOnExit
  >
```